pyproject.toml

```diff
@@ -30,12 +30,14 @@ pexpect = "^4.8.0"
 python-openstackclient = "^6.2.0"
 awscli = "^1.27.100"
 kfp = "==2.7.0"
+kfp-kubernetes = "^1.2.0"
```

Comment:
Why this was added? What problem we solve with this dependency ?

Reply:
It's used by iris_pipeline_disconnected.py to read values from the ConfigMap and store them in the task as env var (I've added now in the description of the PR)